addons/l10n_pl/models/account_move.py

```diff
@@ -16,3 +17,9 @@ class AccountMove(models.Model):
         string='B_MPV_Prowizja',
         help="Supply of agency and other services pertaining to the transfer of a single-purpose voucher",
     )
+
+    def action_post(self):
+        "Validation to avoid having credit notes with more than the invoice"
+        if self.company_id.account_fiscal_country_id.code == 'PL' and self.reversed_entry_id and self.reversed_entry_id.amount_total < self.amount_total:
```

Comment:
You should also check the move type (doing != 'entry' is enough). Indeed, entries can also have a value in reversed_entry_id (when you reverse a misc operation). I checked, and your error message also triggers when doing that ; I don't think we want that, as those are not invoices

Reply:
Done

Reply:
Done